src/components/Button/Button.tsx

```diff
@@ -36,12 +11,10 @@ interface Props {
 
 const Button = (props: Props) => {

```

Comment:
why not Destructuring it like that
const Button = ({color, size, icon}: IButtonProps) => {

Reply:
done